docs/computing/running/greasy.md

```diff
@@ -0,0 +1,120 @@
+
+GREASY was originally developed at BSC. At CSC we use the GREASY version that includes the extensions developed at CSCS. 
+For detailed documentation please check:
+
+   * [GREASY manual (at BSC)](https://github.com/BSC-Support-Team/GREASY/raw/master/doc/greasy_userguide.pdf)
+   * [GREASY instructions (at CSCS) ](https://user.cscs.ch/tools/high_throughput/)
+ 
+!!! Note
+    You should not use GREASY to run MPI parallel tasks. GREASY is not able to manage MPI jobs effectively.
+## GREASY Task lists
+
+Job scheduling with GREASY is based on task lists that have one task (command) in one row. In the simplest approach,
+the task list is just a file containing the commands to be executed. For example, analyzing 200 input files with program _my_prog_ 
+could be described with task list containing 200 rows:
+```text
+my_prog < input1.txt > output1.txt
+my_prog < input2.txt > output2.txt
+my_prog < input3.txt > output3.txt
+...
+my_prog < input200.txt > output200.txt
+```
+
+If needed, you can define **dependencies** between jobs with the `[#line number#]`
+syntax. For example, if you would like to merge the output files of the previous 
+example into a one file, you could add one more row to the task list:
+
+```text
+my_prog < input1.txt > output1.txt
+my_prog < input2.txt > output2.txt
+[#1#] my_prog < input3.txt > output3.txt
+...
+my_prog < input200.txt > output200.txt
+[#1-200#] cat output* > all_output
+```
+In the last line, the definition:
+[#1-200#] ensures that the _cat_ command will be executed only after the tasks from rows 1-200 have finished.
+In addition _[#1#]_ in the third row defines that this task is started only when the first task has been executed. 
+
+By default, all tasks are executed in the directory where the task list processing 
+is launched, but you can add task specific execution directories to the task list with:
+
+```text
+[@ /path/to/folder_task1/ @] my_prog < input1.txt > output1.txt
+[@ /path/to/folder_task2/ @] my_prog < input2.txt > output2.txt
+[@ /path/to/folder_task3/ @] my_prog < input3.txt > output3.txt
+...
+[@ /path/to/folder_task200/ @] my_prog < input200.txt > output200.txt
+```
+!!! Note 
+    You should not include _srun_ to your tasks as GREASY will add it when it executes the tasks. 
+    Please check the [GREASY user guide](https://github.com/BSC-Support-Team/GREASY/raw/master/doc/greasy_userguide.pdf) for more detailed 
+    description of the task list syntax.
+
+## Executing a task list
+
+To use GREASY in Mahti or Puhti, load the GREASY module:
+```text
+module load greasy
+```
+The easiest way to submit a GREASY task list for execution, is to use command _sbatch-greasy_.
+This command automatically creates a batch job file for your task list and submits it:
+```text
+sbatch-greasy tasklist
+```
+The command above will ask you to define the information needed to construct a batch job for the GREASY run. 
+The parameters include: 
+   1. number of cores each task will use (`-c`)
```

Comment:
```suggestion
   1. number of cores **each** task will use (`-c`)
```
testing the suggestion functionality in github and highlighting that this means each task